chrome/content/zotero/elements/attachmentBox.js

```diff
@@ -33,8 +33,10 @@
 			<collapsible-section data-l10n-id="section-attachment-info" data-pane="attachment-info">
 				<html:div class="body">
 					<attachment-preview id="attachment-preview" tabindex="0"/>
-					<label id="url" crop="end" tabindex="0"
-						ondragstart="let dt = event.dataTransfer; dt.setData('text/x-moz-url', this.value); dt.setData('text/uri-list', this.value); dt.setData('text/plain', this.value);"/>
+					<html:div style="display: grid;">
```

Comment:
Why is this necessary?

Reply:
The `lable` will overflow the parent element to the max length of text dramatically if inside non-grid layout parent element